### PR TITLE
Implemented the remove (bulk delete from user) command

### PR DESCRIPTION
**Pull request by aosync**
Implemented the remove command as described in #3  

`.remove <amount> <@user>` deletes the `amount` amount of messages from the user.
Because discord has a fetching messages limit of 100, and that we are targetting a particular user, I chose to iterate several number of times (up to 10 times) through the ranges [-100n-100; -100n] of previous messages until it gets through the required amount of messages from the particular user. If after 10 iterations, the bot cannot find the required amount of messages, it will just give up and delete the ones it has yet found.

The iterations also stop once it gets to a message that's older than 14 days since bulk delete do not work for such old messages.

### CODE REVIEW

**Comment by aosync** [comment]
Don't merge yet, once I can, I will commit the use of a label instead of that messy `breakFromOuter`.

**Comment by aosync** [comment]
Ok I think I am done. Tell me if there are still things to do.

File: command/command.go [hunk 1]
```diff
@@ -64,6 +64,10 @@ var Commands = map[string]Command{
 		Usage: descUsage,
 		Help:  "Sets or clears your description, displays it with fetch",
 	},
+	"remove": moderatorOnly(Command{
+		Exec: remove,
+		Help: removeUsage,
```
Comment (mlvzk) [inline command/command.go]:
```suggestion
		Usage: removeUsage,
```

**Review by mlvzk — CHANGES_REQUESTED** [review]

**Comment by mlvzk** [comment]
Merged, thank you.

**Pull request merged**